sdk/storage/storage-datalake/rollup.config.js

```diff
@@ -24,8 +24,7 @@ const config = {
  * regenerated.
  */`
   },
-  plugins: [
-    nodeResolve({ module: true })
-  ]
+  preserveSymlinks: false,
+  plugins: [nodeResolve({ module: true })]
```

Comment:
I'm guessing this rollup config is for a browser bundle rather than for Node? If so, this probably needs to be:

```suggestion
  plugins: [
    nodeResolve({
      mainFields: ["module", "browser"],
      preferBuiltins: false
    })
  ]
```

Reply:
I am not sure about that, i didn't see a "browser" folder within the package

Reply:
Based on its similarity to other Browser configs, I believe that's the intention.

Reply:
Ok